packages/lib/src/form-fields/multiselect.wc.svelte

```diff
@@ -0,0 +1,101 @@
+<svelte:options
+  customElement={{
+    tag: 'jp-multiselect',
+    shadow: 'none',
+    extend: (customElementConstructor) => {
+      return class extends customElementConstructor {
+        static formAssociated = true;
+
+        constructor() {
+          super();
+          this.attachedInternals = this.attachInternals();
+        }
+      };
+    }
+  }}
+/>
+
+<script lang="ts">
+  import { createEventDispatcher, onMount } from 'svelte';
+
+  export let attachedInternals: ElementInternals;
+
+  export let options: Array<{ label: string; selected: boolean; disabled?: boolean }> = [];
+
+  export let minSelects: number = 0;
+  export let maxSelects: number | null = null;
+  export let required: boolean = false;
+  export let id: string | null = null;
+  export let name: string | null = null;
+  export let placeholder: string = '';
+  export let value: string = '';
+
+  export let open: boolean = false;
+
+  export const getValue = () => options.filter((el) => el.selected).map((el) => el.label);
+
+  const dispatch = createEventDispatcher();
+
+  $: {
+    if (options.filter((el) => el.selected).length < minSelects) {
```

Comment:
Let's do the filtering on top and store it in a variable instead of doing it again in each condition.